recipe_scrapers/nrkmat.py

```diff
@@ -0,0 +1,50 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._exceptions import SchemaOrgException
+from ._grouping_utils import group_ingredients
+
+
+class NRKMat(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "nrk.no"
+
+    def author(self):
+        return (
+            self.soup.find("span", {"class": "caption-cite"})
+            .getText()
+            .removeprefix("Kilde: ")
```

Comment:
Would you be OK with changing this to a `.replace` method call?  At the moment we [continue to support Python 3.8](https://github.com/hhursev/recipe-scrapers/blob/e944acc588cbd0ce6d7d3afdbb60cc0a04a65b2a/pyproject.toml#L28), and this is a [Python 3.9 feature](https://docs.python.org/3/library/stdtypes.html?highlight=removeprefix#str.removeprefix).  Replacement seems like it'll be _slightly_ less accurate in theory, but in practice I doubt it'll cause any problems.

Reply:
fixed